src/Resources/assets/controllers/datetime_controller.js

```diff
@@ -3,9 +3,15 @@ import {easepick, KbdPlugin, TimePlugin} from '@easepick/bundle';
 import easepickStyle from '!!raw-loader!@easepick/bundle/dist/index.css'
 
 export default class extends Controller {
+    /**
+     * @param {String} lang - The language to use within the date picker (translation of months and days)
+     * @param {String} format - The format passed from the server to correctly parse the date
+     */
```

Comment:
I don't think that's the correct annotation format. That's for function parameters. These annotations should just go above the property itself, as you usually would

```js
const example = {
  /**
   * Documentation for property 1
   */
  property1: String
}
```